### DISCUSSION TITLE
Radio Frequencies

### DISCUSSION THREAD

**Discussion by Cornfed14** [Khopa/dcs_liberation#2016] [Q&A] [answered]
Hi I'm trying to start a campaign with the A10 and on the kneeboard it only shows UHF frequencies for my JTAC, ATC, etc. is there a way to change it so that it shows am/fm freqs as well? Thanks.

edit: answered on Discord : 

"Starfire — Today at 12:47 PM
This is due to an old bug where preseting the radio on the A-10 to freqs on the other radios generated an error. So all freqs were generated on a single radio as a workaround. A switch back to using multiple radios is planned for 6.0"

**Comment by Starfire13** [comment, marked as answer]
This is already on the to-do list for 6.0. https://github.com/dcs-liberation/dcs_liberation/issues/1973
